parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -134,8 +146,14 @@ oc create configmap node-config-infra --namespace openshift-node --from-file=nod
 
 # must start ${SERVICE_TYPE}-node after master is fully up and running
 # otherwise the implicit dns change may cause master startup to fail
-systemctl enable ${SERVICE_TYPE}-node.service
-systemctl start ${SERVICE_TYPE}-node.service &
+mv /etc/origin/node/bootstrap.kubeconfig.orig /etc/origin/node/bootstrap.kubeconfig
+
+# Currently the SDN requires node-config.yaml file to be the config file it uses instead of bootstrap-node-config.yaml
+cp /tmp/bootstrapconfigs/master-config.yaml /etc/origin/node/node-config.yaml
+
+sed -i 's#^CONFIG_FILE=.*$#CONFIG_FILE=/etc/origin/node/node-config.yaml#' /etc/sysconfig/${SERVICE_TYPE}-node
```

Comment:
Instead of doing the extra sed why not copy master-config.yaml to the correct path?